middle_end/flambda/inlining/inlining_decision.ml

```diff
@@ -281,6 +281,11 @@ module Call_site_decision = struct
       Format.fprintf fmt "the@ call@ has@ an@ [@@inline always]@ attribute"
     | Attribute_unroll n ->
       Format.fprintf fmt "the@ call@ has@ an@ [@@unroll %d]@ attribute" n
+    | Definition_says_inline ->
+      Format.fprintf fmt "this@ function@ was@ decided@ to@ be@ always@ \
+                          inlinable@ on@ its@ definition@ site (annotated@ by@ \
```

Comment:
```suggestion
                          inlined@ on@ its@ definition@ site (annotated@ by@ \
```

Reply:
I think "inlinable" is correct here, but the wording is maybe not clear.  How about: "it was decided at the definition site that the function should always be inlined"?